### DISCUSSION TITLE
Speech recognition doesn’t work on android browser.

### DISCUSSION THREAD

**Discussion by Sheharyar-Khan401** [deepgram/community#987] [General help]
Hi all, i've implemented deepgram for one of my web application. It is working for every browser but for android browsers speech to text conversion is not working. Can someone help here ?

**Comment by deepgram-community** [comment]
Hey there! It looks like you haven't connected your GitHub account to your Deepgram account. You can do this at https://community.deepgram.com - being verified through this process will allow our team to help you in a much more streamlined fashion.

**Comment by deepgram-community** [comment]
It looks like we're missing some important information to help debug your issue. Would you mind providing us with the following details in a reply?

- The programming language you are working in (e.g. JavaScript, Python).
- A request ID that triggered your error or issue.

**Comment by deepgram-community** [comment]
Sorry we don't have expertise with Android Browsers directly but I can leave this open and tag it with `help wanted` to see if anyone in our community might.

> ![avatar](https://avatars.slack-edge.com/2023-02-20/4855338233600_fad9174023ee3f640ce1_32.png) This message was sent by John Vajda from Deepgram, via our community automation.
